src/Sylius/Bundle/ApiBundle/Converter/CommandFieldItemIriToIdConverter.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Converter;
+
+final class CommandFieldItemIriToIdConverter implements CommandFieldItemIriToIdConverterInterface
```

Comment:
```suggestion
final class CommandFieldItemIriToIdentifierConverter implements CommandFieldItemIriToIdentifierConverterInterface
```
to make it looks more generic

Reply:
What is more, this service is not a converter, it is an object to store command and field